examples/bridge-app/bridge-common/bridge-app.matter

```diff
@@ -285,6 +281,8 @@ client cluster AccessControl = 31 {
   readonly attribute command_id acceptedCommandList[] = 65529;
   readonly attribute attrib_id attributeList[] = 65531;
   readonly attribute bitmap32 featureMap = 65532;
+  readonly attribute bitmap32 featureMap = 65532;
+  readonly attribute int16u clusterRevision = 65533;
```

Comment:
These appear again because these are global attributes and enabled for client and server. Therefore appear twice which makes sense.

Reply:
These attributes do not exist on client clusters in Matter, so no, this does not make sense, nor did the thing here before.  But also, this is IDL for just the client cluster, so why would things appear twice here?

Reply:
Made changes for this. There is a different diff now.